src/app/contract/contractParser.js

```diff
@@ -128,7 +88,7 @@ var formatGasEstimates = function (data) {
   var ret = {}
   var fun
   if ('creation' in data) {
-    ret['Creation'] = gasToText(data.creation[0]) + ' + ' + gasToText(data.creation[1]) + '\n'
+    ret['Creation'] = data.creation
```

Comment:
The new creation object is actually more complex than being a single number:
```
            creation: {
              codeDepositCost: "420000",
              executionCost: "infinite",
              totalCost: "infinite"
            },
```

Do you account for that in code outside of this function?

Reply:
the other place where gas cost is used is for static analysis and yes `infinite` is managed.

Reply:
I mean before this `ret['Creation']` was a string and now it is an object.

Reply:
yes that is ok